lib/printer/hm_a300l.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/services.dart';
+import 'package:flutter_label_printer/flutter_label_printer_platform_interface.dart';
+import 'package:flutter_label_printer/printer/printer_interface.dart';
+import 'package:flutter_label_printer/printer_search_result/bluetooth_result.dart';
+import 'package:flutter_label_printer/printer_search_result/printer_search_result.dart';
+import 'package:flutter_label_printer/src/exception_codes.dart';
+
+/// Interface a Hanyin (HPRT) HM-A300L printer.
+class HMA300L extends PrinterInterface {
```

Comment:
a bit hard to understand what this class is about. how about rename to `HMA300LPrinter`?